RFunction.R

```diff
@@ -12,294 +13,350 @@ library('units')
 
 ########
 ## ToDo: check if uploaded file is .rds of csv, as in the cloud-upload-app, this will also give freedom of name of file
-## ToDo: find correct way of doing this: names(new1) <- make.names(names(new1),allow_=TRUE)
+## ToDo: find correct way of doing this: names(rds_data) <- make.names(names(rds_data),allow_=TRUE)
 #######
 
-rFunction  <-  function(data=NULL, 
-                        time_col="timestamp", 
-                        track_id_col="individual-local-identifier", 
-                        track_attr="",
-                        coords="location-long,location-lat",
-                        crss=4326, ...){ 
-  
-  # rds file # works :)
-  fileName1 <- paste0(getAppFilePath("rdsFile_ID"), "data.rds") #default is NULL
-  logger.info(paste("Reading file", fileName1,"of size:", file.info(fileName1)$size,"."))
-  new1 <- readRDS(fileName1)
-  
-  if(is.na(crs(new1))){
-    new1 <- NULL
+UNSTANDARDIZED_COLS <- c("tag.local.identifier", "sex")
```

Comment:
was running into issue while testing where sometimes these columns were characters and sometimes other types. Coerce to character if un-matching types between data sets.